src/main/java/org/myrobotlab/service/Pir.java

```diff
@@ -18,25 +17,23 @@
 
   private static final long serialVersionUID = 1L;
 
-  boolean isActive = false;
+  String controllerName;
+
+  /**
+   * yep there are 3 states to a binary sensor true/false .... and unknown - we
+   * start with "unknown"
+   */
+  Boolean active = null;
 
   boolean isEnabled = false;

```

Comment:
i know it's not changed in this PR.. but "isEnabled" should be renamed to just "enabled"